std/net/curl.d

```diff
@@ -3480,6 +3533,16 @@ struct FTP
     {
         p.curl.set(CurlOption.infilesize_large, to!curl_off_t(len));
     }
+
+    /**
+       Get the various timings like name lookup time, total time, connect time etc.
+       The timed category is passed through the timing parameter while the timing
+       value is stored at val. The value is usable only if res is equal to CurlError.ok.
+    */
+    CurlCode getTiming(CurlInfo timing, ref double val)
+    {
+        return p.curl.getTiming(timing, val);
```

Comment:
I am certainly missing something, but I can't seem to find the FTP unittests. All unittests only test for the HTTP methods and classes - I see that there is a HTTP server implemented in a unittest but I do not see any ftp implemented server. On the other hand the FTP code seem to be covered, except for my method. I have no idea how that code is covered...